### PR TITLE
I am hoping that this works.

### PR DESCRIPTION
**Pull request by jcrowson12MSU**
Earlier, I lost everything when I tried to commit and merge my revision.

I adopted most of Sean's comments in my revision.
- For his comment about my Reference Information section, I rewrote this section, but focused on making the images more impactful rather than the text. I also restructured this section. Instead of alterating between text and images. I have a numbered list discussing all of the pictures and then all of the pictures are after the number list. I feel like this flows much better.
- For his comment on the generic binary search algorithm, I added a literate block instead of a docstring. I also removed some redundant comments.
- For his comment about documenting the left and right bounds in my code, I restructured the way that I used literate programming and added some more details.
- For his comment about computing the mid point in the code, I added details to the generic algorithm to address this.

### CODE REVIEW

**Review by bjones1 — CHANGES_REQUESTED** [review]
- Delete the .DS_Store file.
- Remove the image from withing your code and keep it in an external file.
- Name the screenshot file something more relevant to its content.
- Put the code in `ShadowOfTheKnight/jic12`.

**Comment by jcrowson12MSU** [comment]
**- Delete the .DS_Store file.**
    - Done
**- Remove the image from within your code and keep it in an external file.**
    - I thought that this was already done.
**- Name the screenshot file something more relevant to its content.**
     - I thought that this was already done. Maybe I misunderstood something.
**- Put the code in ShadowOfTheKnight/jic12.**
    - Do you mean a folder in the outer most directory named or ShadowOfTheKnight/jic12 do you mean that there should be a directory named ShadowOfTheKnight and another directory inside of it named jic12?

File: ShadowOfTheKnight/jic12/ShadowsOfTheKnightSolution_reviewed_by_SBC.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 # # Shadows of the Knight Solution
```
Comment (bjones1) [inline ShadowOfTheKnight/jic12/ShadowsOfTheKnightSolution_reviewed_by_SBC.py:1]: Delete this file -- I only want the final version (which you're already provided).

**Review by bjones1 — CHANGES_REQUESTED** [review]

**Comment by bjones1** [comment]
> - Do you mean a folder in the outer most directory named or ShadowOfTheKnight/jic12 do you mean that there should be a directory named ShadowOfTheKnight and another directory inside of it named jic12?

The second -- your PR is correct (except for one requested deletion).

**Comment by bjones1** [comment]
Looks good!

**Pull request merged**